recipes/dlib/all/conanfile.py

```diff
@@ -33,7 +34,10 @@ class DlibConan(ConanFile):
         "with_png": True,
         "with_sse2": "auto",
         "with_sse4": "auto",
-        "with_avx": "auto"
+        "with_avx": "auto",
+        "with_openblas": True,
+
+        "openblas:shared": False
```

Comment:
we usually don't change default options of dependencies

Reply:
@steromano87 why do you need shared? Does it work with shared only or is a build preference?

Reply:
@uilianries I set the shared option only as a build preference, but it should work also with static library dependency. I took inspiration from other libraries where all dependencies are declared static (like OpenCV).